### DISCUSSION TITLE
Map Homekit "Stopping" state (Active=0 and InUse=1) on Finished and Cancelled

### DISCUSSION THREAD

**Discussion by QuickSander** [QuickSander/homebridge-mieleathome#8] [Ideas]
Currently the mapping is as follows:

Miele State | HomeKit InUse | HomeKit Active | Displayed in Home
------------|----------------|----------------|-------------------
  Off = 1       | 0                         | 0                        | Off
  ProgramSelected = 3 | 0      | 1                         | Starting
  WaitingToStart = 4 | 0          | 1                         | Starting
  InUse = 5 | 1                          | 1                         | Running
  Finished = 7 | 0                     | 1                         | Starting
  Cancelled = 9 | 0                  | 1                         | Starting 

Finished / Cancelled could be mapped as follows instead:

Miele State | HomeKit InUse | HomeKit Active | Displayed in Home
------------|----------------|----------------|-------------------
  Off = 1       | 0                         | 0                        | Off
  ProgramSelected = 3 | 0      | 1                         | Starting
  WaitingToStart = 4 | 0          | 1                         | Starting
  InUse = 5 | 1                          | 1                         | Running
  Finished = 7 | 1                     | 0                         | Stopping
  Cancelled = 9 | 1                  | 0                         | Stopping 

@MichelRabozee Would this be an improvement, or would it break your automation possibilities?

**Comment by MichelRabozee** [comment]
Hi, no problem with my automation (in Eve, you can act both on inUse and active), and yes, having it like the new mapping should be an improvement.
BTW, you seemed a while ago to say that it was not allowed to have inUse=1 with active=0 when I mentioned the Pause status (https://github.com/QuickSander/homebridge-mieleathome/issues/1#issuecomment-760153960).

**Reply by QuickSander** [reply]
> you seemed a while ago to say that it was not allowed to have inUse=1 with active=0 when I mentioned the Pause status

True. Apparently based on a wrong experience. I was triggered by the later bug you discovered with the inuse/active logic that showed "arrêt" for you. Intuitively for me it is still a strange interface, but I have to get used to it.

**Reply by QuickSander** [reply]
I recall what I had in mind when writing it was: when active=0 HomeKit does _not_ show the remaining time. Indirectly related to this new feature, but doesn't prevent this feature.

**Comment by MichelRabozee** [comment]
OK, I understand, and it is logically no issue for the state "stopping" to not display remaining time.
